build.gradle

```diff
@@ -324,6 +257,8 @@ configure(javaProjects) {
         jacoco {
             enabled = jacocoEnabled
             append = false
+            // Exclude the internal dependencies from the coverage report.
```

Comment:
It seems like we shouldn't be running coverage on the shaded jars in the first place